### DISCUSSION TITLE
Questions about loading

### DISCUSSION THREAD

**Discussion by MJiaHao** [Anura3D/Anura3D_OpenSource#137] [Q&A] [answered]
I want to simulate earthquakes by applying equal force time histories to different depths of the soil. As shown in the following picture. I used GID for pre-processing and did not find an option to apply node force in the loading interface, only online or on the surface. Could anura3D be loaded by applying force on the node? Thank you very much.

![7_ F@PFGK%JFT{`55F)}N{T](https://github.com/Anura3D/Anura3D_OpenSource/assets/116955023/b7f76b3d-e8ff-4043-9a49-8c95d14798a0)
![@GE_HKSM1HGMIL{54}U%2_B](https://github.com/Anura3D/Anura3D_OpenSource/assets/116955023/71054aac-a476-49ad-b573-98df8cd2e5b3)

**Comment by luisez1988** [comment]
@AlsardiVirginiaTech, could you help us here?

**Comment by AlsardiVirginiaTech** [comment, marked as answer]
> I want to simulate earthquakes by applying equal force time histories to different depths of the soil. As shown in the following picture. I used GID for pre-processing and did not find an option to apply node force in the loading interface, only online or on the surface. Could anura3D be loaded by applying force on the node? Thank you very much.
> 
> ![7_ F@PFGK%JFT{`55F)}N{T](https://user-images.githubusercontent.com/116955023/273440609-b7f76b3d-e8ff-4043-9a49-8c95d14798a0.png) ![@GE_HKSM1HGMIL{54}U%2_B](https://user-images.githubusercontent.com/116955023/273440612-71054aac-a476-49ad-b573-98df8cd2e5b3.png)

Thanks for your question. In A3D, we have traction boundary conditions that applies stress on a load or a surface. Based on the tributary area, we calculate the force on the node. If you want to apply a force on a node, you need to identify the particular node numbers where you want to apply the force, find the degree of freedom associated with it, and hardcode in the value of the force in MS Visual Studio. If you have further questions, please feel free to reply on this discussion.

**Comment by kuailebozai** [comment]
Hello @AlsardiVirginiaTech, I would also like to simulate earthquakes by applying equal force time histories, your description is very challenging for a non-computer science student like me, I would like you to have a detailed description of the problem, very much looking forward to your reply, thanks!
